examples/cmd/nse/icmp-responder-nse/nse.go

```diff
@@ -72,6 +79,11 @@ func main() {
 		defer func() { _ = nsmEndpoint.Delete() }()
 	}
 
+	if *update {
+		<-time.After(10 * time.Second)
```

Comment:
Why we wait here 10 seconds?

Reply:
To be sure that client has connected to NSM and has started monitoring connection update events.

Reply:
We need to send immediately and handle on init state properly if state are changed we have updated on NSC.

Reply:
Done : 43e3fdb